data/cy/lms_2.json

```diff
@@ -9853,7 +9853,7 @@
                         "questions": [{
                             "id": "no-primary-why-uk-question",
                             "titles": [{
-                                    "value": "Beth oedd prif reswm <em>{{ [answers['primary-household-member-first-name'][group_instance], answers['primary-household-member-last-name'][group_instance]] | format_household_name}}</em>  dros ddod i fyw yn y Deyrnas Unedig yn {{ answers['no-primary-first-arrive-in-uk-answer'][group_instance]|format_date }} ?",
+                                    "value": "Beth oedd prif reswm <em>{{ [group_instances[group_instance_id]['other-household-member-first-name-no-primary'], group_instances[group_instance_id]['other-household-member-last-name-no-primary'], group_instances[group_instance_id]['student-household-member-first-name-no-primary'], group_instances[group_instance_id]['student-household-member-last-name-no-primary']] | format_household_name }}</em>  dros ddod i fyw yn y Deyrnas Unedig yn {{ answers['no-primary-first-arrive-in-uk-answer'][group_instance]|format_date }} ?",
```

Comment:
Do we need to add this change to the spreadsheet to keep everything up to date?

Reply:
Done.